src/openrct2/ride/Vehicle.cpp

```diff
@@ -72,15 +72,15 @@ uint8_t _vehicleF64E2C;
 Vehicle* _vehicleFrontVehicle;
 CoordsXYZ unk_F64E20;
 
-static constexpr const OpenRCT2::Audio::SoundId byte_9A3A14[] = {
+static constexpr const OpenRCT2::Audio::SoundId screams_0[] = {
```

Comment:
```suggestion
static constexpr const OpenRCT2::Audio::SoundId _screamSounds0[] = {
```
I think this would better reflect our code style